src/app/(proper_react)/(onboarding)/user/welcome/View.module.scss

```diff
@@ -0,0 +1,197 @@
+      padding-inline: $spacing-xl;
+    }
+  }
+
+  .stepsWrapper {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    @media screen and (min-width: $screen-md) {
+      height: 33%;
+    }
+  }
+
+  .stepsWrapper,
+  .contentWrapper {
+    padding-inline: $spacing-lg;
+
+    @media screen and (min-width: $screen-md) {
+      padding-inline: $spacing-2xl;
+    }
+
+    @media screen and (min-width: $screen-lg) {
+      padding-inline: $layout-xl;
+    }
+  }
+
+  @media screen and (max-width: $screen-md) {
+    .contentWrapper {
+      $backgroundImageHeight: 200px;
+      padding-top: $backgroundImageHeight + $spacing-lg;
+      background-image: url("./images/Scanning.svg");
+      background-repeat: no-repeat;
+      background-position-x: center;
+      background-size: auto $backgroundImageHeight;
+    }
+  }
+}
+
+@media screen and (min-width: $screen-md) {
+  .steps {
+    $stepGap: $spacing-2xl;
+    padding-inline-start: 0;
+    list-style-type: none;
+    display: flex;
+    justify-content: space-evenly;
+    gap: $stepGap;
+    max-width: $content-sm;
+    position: relative;
+
+    li {
+      $iconDiameter: 27px;
+      $iconBorderWidth: 4px;
+      $connectingLineHeight: 4px;
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      gap: $spacing-xs;
+      text-align: center;
+      font: $text-body-xs;
+      font-weight: 600;
+      color: $color-grey-30;
+
+      img {
+        background-color: $color-grey-30;
+        padding: 5px;
+        margin-block: $iconBorderWidth;
+        width: $iconDiameter;
+        height: $iconDiameter;
+        border-radius: 100%;
+        box-sizing: content-box;
+        // Overlap the `::before` (the horizontal line)
+        z-index: 1;
+      }
+
+      &:nth-child(n + 2)::before {
+        content: "";
+        background-color: $color-grey-30;
+        width: calc(25% + $stepGap);
+        display: block;
+        height: $connectingLineHeight;
+        position: absolute;
+        top: calc(
+          ($iconDiameter + $connectingLineHeight) / 2 + $iconBorderWidth
+        );
+      }
+      &:nth-child(2)::before {
+        left: calc(25% - $iconDiameter / 2 - $stepGap / 2);
+      }
+      &:nth-child(3)::before {
+        left: 50%;
+      }
+
+      &[aria-current="step"] {
+        img {
+          margin-block: 0;
+          border: $iconBorderWidth solid $color-purple-10;
+        }
+      }
+
+      &.isCompleted,
+      &[aria-current="step"] {
+        color: $color-purple-70;
+
+        &::before {
+          background-color: $color-purple-10;
+        }
+
+        img {
+          background-color: $color-purple-70;
+        }
+      }
+    }
+  }
+}
+@media screen and (max-width: calc($screen-md - 1px)) {
+  .steps {
+    padding-inline-start: 0;
+    list-style-type: none;
+    display: flex;
+    justify-content: space-evenly;
+    gap: $spacing-sm;
+
+    li {
+      img {
+        height: 0;
```

Comment:
What does a `0` height on an `<img>` tag do?

Reply:
Added a comment in b76cc4b0c, let me know if that doesn't sufficiently answer it. (Or if you have concerns with the approach - the images are fairly light-weight

Reply:
Nit: We could give the images the following CSS attributes as well so they can’t be interact with:
```
pointer-events: none;
user-select: none;
```

Reply:
Oh I love that, that's just the bit of polish that distinguishes websites from apps! 74300a3c6

Probably also a good idea for the images on large screens: 182340a27

And in fact, the illustration in the dialog too: 2cf4dcc79